packages/mendel-generator-prune/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "mendel-generator-prune",
-  "version": "3.0.0",
-  "description": "Mendel post-generator that cleans up dangling dependencies and mangles `expose` for shorter names.",
+  "version": "3.0.1",
+  "description": "Mendel post-generator that cleans up dangling dependencies and normalizes `expose` fields.",
```

Comment:
suggestion to not use `expose` because this is a browserify term. We don't expose, we just never hide it anyway 😄